### DISCUSSION TITLE
Team-A wiki

### DISCUSSION THREAD

**Discussion by FrankTeusink** [AmpersandTarski/Ampersand#1425] [General]
I would like a easy (web) location for team-related documentation.

Containing:
* Week schedule of members
* Holiday planning
* Check lists and To Do's for new members that are not specific to individual repositories
* ...

Not containing:
* Architecture
* Check lists w.r.t. work on specific repositories/products

Candidates:
* Activate Wiki feature on RAP repository
* Create A-Team repository with Wiki
* Note: It is (to my knowledge) not possible to add Wiki's on organization level (AmpersandTarski)
* Ordina Teams team" Project A & NUT-Vluchtelingen opensource 
* ...

**Comment by stefjoosten** [comment]
I think we should use MS-Teams for this purpose, simply because [every team member is already on it](https://teams.microsoft.com/l/team/19%3ayM9P1tFiWIADqDUbLDyX7ksB1Oavi04StkxyS6grh7A1%40thread.tacv2/conversations?groupId=09b86f1c-3ba6-411d-9b16-f0915eb2ed8a&tenantId=a254b169-0a6b-47f9-af4c-169704421c2e). In Teams, we should do everything that is not supposed to be consolidated for posterity in an open source directory. That includes personal things, but also scrum boards and other transient stuff.
